proto/spire/plugin/agent/nodeattestor/v1/nodeattestor.proto

```diff
@@ -1,37 +1,50 @@
-/** Responsible for attesting the physical nodes identity.  The plugin
-will be responsible to retrieve an identity document or data associated
-with the physical node.  This data will be used when attesting to the server.
-*/
-
 syntax = "proto3";
-package spire.agent.nodeattestor;
-option go_package = "github.com/spiffe/spire/proto/spire/agent/nodeattestor";
+package spire.plugin.agent.nodeattestor.v1;
+option go_package = "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/agent/nodeattestor/v1;nodeattestorv1";
 
-import "spire/common/common.proto";
-import "spire/common/plugin/plugin.proto";
+service NodeAttestor {
+    // FetchAttestationData retrieves the attestation payload and optionally
+    // participates in challenge/response attestation mechanics.
+    //
+    // The attestation flow is as follows:
+    // 1. SPIRE agent opens up a stream to the plugin via FetchAttestationData.
```

Comment:
nit: the convention that we have is to uppercase the first letter for server/agent when we name "SPIRE Server" or "SPIRE Agent" and lowercase when we refer to them as "the agent" or "the server". There are several instances of this in this file.

Reply:
Thanks for catching that. I'll fix up other PRs.